File: examples/actions/node-echo/handler.js
// @ryvus-sdk/api
import { apiAction } from "../../../sdk/node/dist/index.js";

export default apiAction((event, context) => {
  console.log("hello from node");
  console.warn("this is a warning");
  console.info("this is some info");
  console.error("this is an error");
  console.trace("this is a trace");

  return {
    message: "Hello from Ryvus Node SDK",
  };
});

File: examples/actions/python-echo/handler.py
import logging
from ryvus import api_action

logger = logging.getLogger(__name__)

@api_action
def handler(event, context):
    print("hello from print")

    logger.debug("debug from python")
    logger.info("info from python")
    logger.warning("warning from python")
    logger.error("error from python")

    return {
        "message": "Hello from Ryvus Python SDK"
    }